server/src/testing/testingRoutes.ts

```diff
@@ -128,5 +128,59 @@ export default function testingHandler(server: FastifyInstance, options, done) {
     console.log(res)
     reply.code(200).send(res)
   })
+  server.get('/genData', async (req, reply) => {
+    // (eg) add data to local firestore emulator for testing purposes
+    try {
+      const team1 = await firestoreDb.collection('teams').add({
+        name: 'test_team',
+        icon: 'test_icon',
+        discordRole: 'test_role',
+        githubTeam: 'test_github',
+        defaultDiscordChannel: 'test_channel' // pass a valid channel id here while testing
+      })
+      await firestoreDb.collection('teams').add({
+        name: 'test_team_2',
+        icon: 'test_icon_2',
+        discordRole: 'test_role_2',
+        githubTeam: 'test_github_2',
+        defaultDiscordChannel: 'test_channel_2'
+      })
+      await firestoreDb.collection('users').add({
+        name: 'test_user_1',
+        githubId: 'github_user_1',
+        discordId: 'discord_user_1',
+        team: team1.id,
+        teamLead: true,
+      })
+      await firestoreDb.collection('users').add({
+        name: 'test_user_2',
+        githubId: 'github_user_2',
+        discordId: 'discord_user_2',
+        team: team1.id,
+        teamLead: false,
+      })
+
+      console.log('database seed was successful');
+    } catch (error) {
+      console.log(error, 'database seed failed');
+    }
+  })
+  server.get('/testcleanup', async (req, reply) => {
+    const targetTeam = await firebaseApi.getTeam('test_role')
```

Comment:
This doesn't clean up the 2nd team

Reply:
didn't get it...the script is supposed to mimic the cleanupcommand which basically takes a team and resets it, it does not handle multiple teams input

Reply:
Ah I see what this is intending - okay to leave as is